lib-impl/src/main/java/org/ergoplatform/appkit/impl/ErgoProverBuilderImpl.java

```diff
@@ -49,34 +46,16 @@ public ErgoProverBuilder withSecretStorage(SecretStorage storage) {
     }
 
     /**
-     * This code uses original cryptographic notation known from the literature, see the following
-     * <a href="https://github.com/ScorexFoundation/sigmastate-interpreter/blob/b3695bdb785c9b3a94545ffea506358ee3f8ed3d/sigmastate/src/test/scala/sigmastate/utxo/examples/DHTupleExampleSpecification.scala#L28">example</a>
-     * to understand the implementation, where this _masterKey belong to Bob
+     * ProveDHTuple is of the form (g, h, u, v) with secret x (and unknown y), where:
+     *   h = g^y
+     *   u = g^x
+     *   v = g^xy
+     *
+     *   NOTE: We can swap x, y and obtain another tuple (g, u, h, v) with secret y (and unknown x).
```

Comment:
Again you can add this crypto terminology in addition to the original names and docs.

Reply:
Agree (I removed the original test by mistake). We need to put it back.